lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -2793,6 +2793,10 @@ void __cdecl _alloca_probe_16()
             {
                 *value = scriptContext->GetLibrary()->GetNull();
             }
+            else
```

Comment:
funcCaller->GetScriptContext() != requestContext here. Marshalling was needed due to this.